exercises/fizz-buzz/README.md

```diff
@@ -0,0 +1,29 @@
+# Fizz Buzz
+
+Print numbers from 1 to n with the following exceptions:
+
+1. When a number is a multiple of 3, then print "Fizz"
+2. When a number is a multiple of 5, then print "Buzz"
+3. When a number is both a multiple of 3 and a multiple of 5, then print "Fizz Buzz"
+
+The output should be a vector of strings, such as, "1" "2" "Fizz" "4" "Buzz" ...
+
+## Installation
+See [this guide](https://exercism.io/tracks/r/installation) for instructions on how to setup your local R environment.
+
+## How to implement your solution
+In each problem folder, there is a file named `<exercise_name>.R` containing a function that returns a `NULL` value. Place your implementation inside the body of the function.
+
+## How to run tests
+Inside of RStudio, simply execute the `test_<exercise_name>.R` script. This can be conveniently done with [testthat's `auto_test` function](https://www.rdocumentation.org/packages/testthat/topics/auto_test). Because exercism code and tests are in the same folder, use this same path for both  `code_path` and `test_path` parameters. On the command-line, you can also run `Rscript test_<exercise_name>.R`.
+
+## Source
+
+"Fizz-Buzz" originated as a children's game that helped children learn to divide. It is now regularly used in coding interviews. 
+
+1. [Rosetta Code](https://rosettacode.org/wiki/FizzBuzz) 
+2. [Code Golf](https://code-golf.io/fizz-buzz)
+
```

Comment:
Not sure if useful to add the link here, but the _actual_ source seems to be https://imranontech.com/2007/01/24/using-fizzbuzz-to-find-developers-who-grok-coding/

Reply:
I saw this link too. I'll remove the others then and just use this one

Reply:
Na, well, it's a bit nasty blog-post to be honest so I'd only _add_ it, with a tounge-in-cheek. Mentioning those other two coding riddle resources is fine with me.